db/migrate/20181225142223_create_posts.rb

```diff
@@ -0,0 +1,17 @@
+class CreatePosts < ActiveRecord::Migration[5.1]
+  def change
+    create_table :posts do |t|
+      t.string :name
```

Comment:
Database constraints(like `NOT NULL`,  are very important). 
Nobody is protected from bad programmers' arms. And even if you add a validation for presence, Rails allows you to skip validations sometimes. 

Please add `null: false` to some columns here and re-run the mogrations